packages/sign_in_with_apple/lib/src/exceptions.dart

```diff
@@ -78,8 +78,8 @@ class UnknownSignInWithAppleException extends PlatformException
 /// An [SignInWithAppleException] which will be thrown in case Sign in with Apple is not supported.
 class SignInWithAppleNotSupportedException implements SignInWithAppleException {
   const SignInWithAppleNotSupportedException({
-    @required this.message,
-  }) : assert(message != null);
+    required this.message,
```

Comment:
Is `required` now a keyword of Dart and you don't need the annotation anymore?

Reply:
Yes, and `required` is stronger: It is an integrated part of the type system. So this change is strongly recommended.